rules/do-you-have-an-endomarketing-strategy-in-your-company/rule.md

```diff
@@ -1,112 +1,61 @@
 ---
 type: rule
-archivedreason: 
 title: Do you have an Endomarketing (Internal Marketing) strategy in your company?
-guid: 7417b806-1540-452b-bcf5-64987e6b3702
 uri: do-you-have-an-endomarketing-strategy-in-your-company
-created: 2019-06-25T04:17:06.0000000Z
 authors:
-- title: Camilla Rosa Silva
-  url: https://ssw.com.au/people/camilla-rosa-silva
-- title: Jerwin Parker Roberto
-  url: https://ssw.com.au/people/jerwin-parker
-  img: https://github.com/SSWConsulting/SSW.People.Profiles/raw/main/Jerwin-Parker/Images/Jerwin-Parker-Profile.jpg
+  - title: Camilla Rosa Silva
+    url: https://ssw.com.au/people/camilla-rosa-silva
+  - title: Jerwin Parker Roberto
+    url: https://ssw.com.au/people/jerwin-parker
+    img: https://github.com/SSWConsulting/SSW.People.Profiles/raw/main/Jerwin-Parker/Images/Jerwin-Parker-Profile.jpg
 related:
-- do-you-have-a-waiting-area-that-reinforces-your-marketing-profile
-- how-to-see-what-is-going-on-in-your-project
-- do-you-use-microsoft-advertising-formerly-known-as-bing-ads
-- branding-do-you-know-when-to-use-an-overlay-on-images
+  - do-you-have-a-waiting-area-that-reinforces-your-marketing-profile
+  - how-to-see-what-is-going-on-in-your-project
+  - do-you-use-microsoft-advertising-formerly-known-as-bing-ads
+  - branding-do-you-know-when-to-use-an-overlay-on-images
 redirects:
-- do-you-have-an-endomarketing-internal-marketing-strategy-in-your-company
-- do-you-have-an-endomarketing-(internal-marketing)-strategy-in-your-company
+  - do-you-have-an-endomarketing-internal-marketing-strategy-in-your-company
+  - do-you-have-an-endomarketing-(internal-marketing)-strategy-in-your-company
+created: 2019-06-25T04:17:06.000Z
+archivedreason: null
+guid: 7417b806-1540-452b-bcf5-64987e6b3702
 
 ---
 
-Endomarketing (also known as  **Internal Marketing)** is one of the most effective strategies to reduce your employee turnover rate and attract new highly qualified professionals to your company. 
-
-
-
-
-
-This strategy is mainly directed at creating an image of a great workplace and making people feel like their current position is a dream job. The prefix ‘endo’ comes from the Greek meaning "inside," therefore Endomarketing in a business environment is targeted at your employees. 
-
-
+Endomarketing (also known as **Internal Marketing**) is one of the most effective strategies to reduce your employee turnover rate and attract new, highly qualified professionals to your company. 
 
+This strategy is mainly directed at creating an image of a great workplace and making people feel like their current position is a dream job. 
```

Comment:
I would suggest changing:
> creating an image

to:
nurturing a feeling

Reply:
```suggestion
This strategy is mainly directed at nurturing a feeling of a great workplace and making people feel like their current position is a dream job. 
```